src/treq/test/test_multipart.py

```diff
@@ -65,7 +59,7 @@ def getOutput(self, producer, with_producer=False):
 
     def newLines(self, value):
 
-        if isinstance(value, text_type):
+        if isinstance(value, str):
             return value.replace(u"\n", u"\r\n")
```

Comment:
And in theory can kill the `u""` prefix perhaps.

Reply:
I guess that if you want to kill py2 for good, you can run https://github.com/PyCQA/modernize and have all the code updated in one go.

Reply:
modernize is for adding py3 support to a py2 project. You probably want pyupgrade

Reply:
I don't view removing `u''` prefixes as worth the diff churn at the moment. That'll happen when adopting Black.